modules/sno-du-enabling-sctp.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/sno-du-deploying-clusters-on-single-nodes.adoc
+
+:_content-type: PROCEDURE
+[id="sno-du-enabling-sctp_{context}"]
+= Enabling Stream Control Transmission Protocol (SCTP)
+
+SCTP is a key protocol used in RAN applications. This `MachineConfig` adds the SCTP kernel module to the node to enable this protocol.
+
+.Procedure
+
```

Comment:
Does this CR exist or does the user need to create?

Reply:
Procedures need steps, not bullet list

Reply:
The user uses the example to create the CR.